documentation/experimental_features.md

```diff
@@ -418,48 +418,3 @@ To encrypt SSH connections using the unsupported algorithm
 > option to shell out to SSH. See [the net-ssh
 > README](https://github.com/net-ssh/net-ssh/#supported-algorithms) for a list of supported
 > algorithms.
-
-## `bolt plan new` command
```

Comment:
We should add some docs about this to the `writing plans` documentation for folks to easily get started, and should maybe also add this to the Getting Started guide as a note for future plan creation.

Reply:
Added sections to Writing (YAML) Plans pages. I'm going to leave out the Getting Started guide for now since Heston is in the process of reworking that.